cypress/e2e/models/migration/applicationinventory/analysis.ts

```diff
@@ -379,7 +379,7 @@ export class Analysis extends Application {
             .contains(this.name)
             .closest(trTag)
             .within(() => {
-                cy.get(effortColumn).should("contain", `${effort}`);
+                cy.get(effortColumn, { timeout: 50 * SEC }).should("contain", `${effort}`);
```

Comment:
This can be removed, if the app takes so much time to reflect the effort after an analysis has completed it is probably due to a bug